confuse.py

```diff
@@ -784,13 +784,21 @@ def check_plain(self):
 Loader.add_constructor('tag:yaml.org,2002:omap', Loader.construct_yaml_map)
 
 
-def load_yaml(filename):
+def load_yaml(filename, loader=Loader):
     """Read a YAML document from a file. If the file cannot be read or
     parsed, a ConfigReadError is raised.
     """
+    if not isinstance(loader, Loader):
+        loader.add_constructor('tag:yaml.org,2002:str',
+                               Loader._construct_unicode)
+        loader.add_constructor('tag:yaml.org,2002:map',
+                               Loader.construct_yaml_map)
+        loader.add_constructor('tag:yaml.org,2002:omap',
+                               Loader.construct_yaml_map)
```

Comment:
I got your point, the problem aren't the constructors, they are very fine (for me).
The problem (for me) is the base SecureLoader you choosed, it avoids the yaml tags !!python/object:__main__.type or other some of the python tags.

Reply:
Right, I understand—but I think if we're going to do this, we should give people the ability to avoid these constructors. Even if it's not for your use case, it seems like an obvious thing someone else might ask for. (And see also the note about mutation above—this behavior could be quite confusing!)

I'm proposing a separate function that just adds those constructors to any loader. For your use case, you would call that before passing the loader into Confuse. This also avoids the need to thread the `override` flag through everywhere.